JointStatePublisher:
    type: valkyrie_translator/JointStatePublisher
    publish_imu_readings: true
    publish_separate_force_torque_readings: true
